scripts/runtests.py

```diff
@@ -0,0 +1,165 @@
+    print("\nRunning pycln...")
+    subprocess.run([sys.executable, "-m", "pycln", path, "--all"])
+    print("\nRunning isort...")
+    subprocess.run([sys.executable, "-m", "isort", path])
+    print("\nRunning Black...")
+    black_result = subprocess.run([sys.executable, "-m", "black", path])
+    if black_result.returncode == 123:
+        print("Could not run tests due to an internal error with Black. See above for details.", file=sys.stderr)
+        sys.exit(black_result.returncode)
+
+    print("\nRunning Flake8...")
+    flake8_result = subprocess.run([sys.executable, "-m", "flake8", path])
+
+    print("\nRunning check_consistent.py...")
+    check_consistent_result = subprocess.run([sys.executable, "tests/check_consistent.py"])
+    print("\nRunning check_new_syntax.py...")
+    check_new_syntax_result = subprocess.run([sys.executable, "tests/check_new_syntax.py"])
+
+    print(f"\nRunning Pyright on Python {_PYTHON_VERSION}...")
+    pyright_result = subprocess.run(
+        [sys.executable, "tests/pyright_test.py", path, "--pythonversion", _PYTHON_VERSION] + _get_strict_params(path),
+        stderr=subprocess.PIPE,
+        text=True,
+    )
+    if re.match(r"error (runn|find)ing npx", pyright_result.stderr):
+        print("\nSkipping Pyright tests: npx is not installed or can't be run!")
+        pyright_returncode = 0
+        pyright_skipped = True
+    else:
+        print(pyright_result.stderr)
+        pyright_returncode = pyright_result.returncode
+        pyright_skipped = False
+
+    print(f"\nRunning mypy for Python {_PYTHON_VERSION}...")
+    mypy_result = subprocess.run([sys.executable, "tests/mypy_test.py", path, "--python-version", _PYTHON_VERSION])
+    # If mypy failed, stubtest will fail without any helpful error
+    if mypy_result.returncode == 0:
+        print("\nRunning stubtest...")
+        if folder == "stdlib":
+            stubtest_result = subprocess.run([sys.executable, "tests/stubtest_stdlib.py", stub])
+        else:
+            stubtest_result = subprocess.run([sys.executable, "tests/stubtest_third_party.py", stub])
+    else:
+        print("\nSkipping stubtest since mypy failed.")
+
+    if sys.platform == "win32":
+        print("\nSkipping pytype on Windows. You can run the test with WSL.")
+    else:
+        print("\nRunning pytype...")
+        pytype_result = subprocess.run([sys.executable, "tests/pytype_test.py", path])
+
+    print(f"\nRunning regression tests for Python {_PYTHON_VERSION}...")
+    regr_test_result = subprocess.run(
+        [sys.executable, "tests/regr_test.py", "stdlib" if folder == "stdlib" else stub, "--python-version", _PYTHON_VERSION],
+        stderr=subprocess.PIPE,
+        text=True,
+    )
+    # No test means they all ran successfully (0 out of 0). Not all 3rd-party stubs have regression tests.
+    if "No test cases found" in regr_test_result.stderr:
+        regr_test_returncode = 0
+        print(colored(f"\nNo test cases found for '{stub}'!"), "green")
```

Comment:
Currently this prints:

```
No test cases found for 'redis'! green
```

```suggestion
        print(colored(f"\nNo test cases found for {stub!r}!", "green"))
```